src/main/scala/battleship/routes/gameActor.scala

```diff
@@ -0,0 +1,84 @@
+package battleship.routes

```

Comment:
I did place it randomly, because I couldn't figure out the logic behind the packages 'routes' and 'game'. Can you elaborate?